recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set version = "2.4.rc3" %}
+{% set source_tag = "2.4-rc3" %}
+{% set build_number = 0 %}
+
+{% set variant = "openblas" %}
+{% set build_number = build_number + 200 %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/{{ source_tag }}.tar.gz
+  sha256: 48a52af8f9da6deef5c01d26a30ba4bdc19929999d6680dab6461b92b5a63bd7
+
+build:
+  number: {{ build_number }}
+  skip: true  # [win and vc<14]
+  features:
+    - blas_{{ variant }}
+
+requirements:
+  build:
+    - cmake >=3.8
+    - {{ compiler('cxx') }}
+
+  host:
+    - python
+    - numpy
+    - blas 1.1 {{ variant }}
+    - openblas
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib
+    - libmatio
+
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - blas 1.1 {{ variant }}
+    - openblas
+    - hdf5 1.8.18|1.8.18.*
+    - zlib
+    - libmatio
+    - libcxx 4.0.*  # [osx]
```

Comment:
Believe this can be dropped as `defaults` adds it explicitly with their compilers.